package.json

```diff
@@ -44,9 +44,11 @@
   },
   "homepage": "https://github.com/multiformats/js-multibase#readme",
   "dependencies": {
-    "bs58": "^3.0.0"
+    "base-x": "1.0.4",
+    "bs58": "^3.0.0",
+    "webpack": "^1.13.2"
```

Comment:
`webpack` does not need to be a dependency. This comes in as a devDep through `aegir` already.